### DISCUSSION TITLE
How to configure the generation of Openshift's RBAC objects in openshift.yml ?

### DISCUSSION THREAD

**Discussion by cvgaviao** [quarkusio/quarkus#44851] [Q&A]
Hello.
I would like to use the openshift.yml that is being generated by my Quarkus application.

Unfortunately, my company's Openshift team doesn't allow us to create service-account (we must use one provided for each team), roles and rolebindings (but we can ask them to create).

So, is there a way to configure the generation of openshift.yaml so it do not add a service-account and rbac objects, but use existent ones ?

[a comment by an automated account is not shown]

**Reply by geoand** [reply]
I don't remember really. @iocanel do you?
